test/rekt/crossnamespace_test.go

```diff
@@ -50,5 +52,7 @@ func TestBrokerTriggerCrossNamespaceReference(t *testing.T) {
 	)
 
 	// brokerEnv.Test(brokerEnvCtx, t, broker.GoesReady(brokerName))
-	triggerEnv.Test(triggerEnvCtx, t, trigger.CrossNamespaceEventLinks(brokerEnvCtx))
+	triggerEnv.Test(triggerEnvCtx, t, trigger.CrossNamespaceEventLinks(brokerEnvCtx, func(name, namespace string) feature.StepFn {
+		return broker.Install(name, broker.WithNamespace(namespace))
```

Comment:
Is there a reason you didn't reuse `broker.GoesReady(name, brokerresources.WithEnvConfig()...)`?